src/lib/dom.generated.d.ts

```diff
@@ -7344,7 +7344,7 @@ interface Event {
 }
 declare var Event: {
     prototype: Event;
-    new(): Event;
+    new(type: string): Event;

```

Comment:
:+1:

This constructor throws if no name is provided, so this parameter should be considered required.